packages/components/src/components/accordion/accordion-styles.ts

```diff
@@ -136,6 +151,10 @@ export const getComponentCss = (
         color: primaryColorDark,
       }),
       display: 'grid',
+      ...(sticky && {
+        position: 'relative',
+        zIndex: zIndexes.indexOf('collapsible'),
```

Comment:
why not simply define `zIndex: 0` (maybe with a comment) then we'd save unnecessary bytes in the bundled chunk?

Reply:
Coming from a Sass background I found it way more intuitive to define such z-index ordering as a list, but you are right, we need to primarily safe bytes and a comment should be fine.